src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -329,7 +329,7 @@ public static Date parseDate (String str) {
 
 	public static Date parseTime (String str) {
```

Comment:
Are there other paths into here besides ```XFormAnswerDataParser.getAnswerData```, such as ```TimeData.cast```, that shouldn’t have the time parsed in this new way? If so, and the path from getAnswerData is the only path that needs to be affected, perhaps getAnswerData should call a new function instead of this one? Such as parseTimeIgnoreTimezone.